ablySpec/RestChannel.swift

```diff
@@ -198,14 +212,203 @@ class RestChannel: QuickSpec {
                         
                         expect(message.content()).toNot(beNil())
                         expect(message.action).to(equal(ARTPresenceAction.Present))
-                        
+
                         // skip the encrypted message for now
-                        if message.payload.encoding?.rangeOfString("cipher") == nil {
+                        if message.payload.encoding.rangeOfString("cipher") == nil {
                             expect(message.content() as? NSObject).to(equal(fixtureMessage["data"].object as? NSObject))
                         }
                     }
                 }
             }
         }
+
+        describe("message encoding") {
+
+            struct TestCase {
+                let value: AnyObject?
+                let expected: JSON
+            }
+
+            let text = "John"
+            let integer = "5"
```

Comment:
Not sure I see the value in this test given it's not an integer, but no point removing either